pxr/imaging/hdSt/materialXShaderGen.cpp

```diff
@@ -285,12 +289,15 @@ HdStMaterialXShaderGen<Base>::_EmitMxSurfaceShader(
         if (outputConnection) {
 
             std::string finalOutput = outputConnection->getVariable();
+#if MATERIALX_MAJOR_VERSION == 1 && MATERIALX_MINOR_VERSION==38
```

Comment:
Worth doing "MATERIALX_MAJOR_VERSION == 1 && MATERIALX_MINOR_VERSION < 39"?

Reply:
I don't think any version of MaterialX prior to 1.38.something was ever used or would compile against the USD codebase, so this seemed more direct, but if you prefer the `<` I have no objections to changing it. 

Maybe `#if MATERIALX_MAJOR_VERSION == 1 && MATERIALX_MINOR_VERSION<=38` is better overall?